src/lightbox/LightboxHeader.js

```diff
@@ -73,9 +73,11 @@ export default class LightboxHeader extends PureComponent<Props> {
               {subheader}
             </Text>
           </View>
-          <Touchable style={styles.rightIconTouchTarget} onPress={onPressBack}>
```

Comment:
On iOS, the "x" button in the header was putting a brand-colored
    rectangle around the icon when it was pressed, which seemed jarring
    and artificial to me.

Hmm, yeah.

I actually reproduce that on Android too, now that I look closely. I think it's less visible there, because the animation that takes us back to the message list is quicker to get it out of view. On Android we get the standard Android "fade down" animation for returning to a previous screen, and on iOS we get the standard iOS "slide out right" animation, and I guess the latter has a slower start.